CHANGELOG.unreleased.md

```diff
@@ -41,3 +41,5 @@ Examples:
   ```
 
 -->
+
+JavaScript: fix edge cases for type cast comment detection ([#5793] by [@yangsu])
```

Comment:
Could you fill in the description and code samples? It's not clear what do edge cases refer to.